app/src/main/java/org/wikipedia/dataclient/donate/Campaign.kt

```diff
@@ -23,12 +25,45 @@ class Campaign(
         return platforms.containsKey(platform)
     }
 
+    fun getIdForLang(lang: String): String {
+        val assetsForLang = getAssetsForLang(lang)
+        if (assetsForLang == null || assetsForLang.id.isEmpty()) {
+            return id
+        }
+        return id + "_" + assetsForLang.id
+    }
+
     fun getAssetsForLang(lang: String): Assets? {
-        return assets[lang]
+        val list = assets[lang].orEmpty()
+        if (list.isEmpty()) {
+            return null
```

Comment:
instead of having a `orEmpty()` on line `37`, can we use a `list.isNullOrEmpty` on line `38`?

Reply:
Sure!